libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -213,7 +213,24 @@ func (m *Manager) GetPids() ([]int, error) {
 		return nil, err
 	}
 
-	return cgroups.ReadProcsFile(dir)
+	var pids []int
+	var readErr error
+	// collect pids from all sub-cgroups
+	filepath.Walk(dir, func(p string, info os.FileInfo, err error) error {
```

Comment:
Nope, I'm not returning it as well. I think I need to return at leas in case of "cgroup.procs"